Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackIntegrationShell.h

```diff
@@ -0,0 +1,9 @@
+#import "SentryBaseIntegration.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryUserFeedbackIntegrationShell : SentryBaseIntegration
```

Comment:
So, in the previous PR, I made `SentryUserFeedbackIntegration` a swift file, before I knew that it isn't currently possible to implement integrations in Swift (AFAICT anyway). But, renaming that file/class so that this objc class could have that name made the diff of that file messy, so for now I'm making this a placeholder name, and I'll do the renames in a separate PR.